zenodo/modules/utils/tasks.py

```diff
@@ -263,3 +270,71 @@ def update_search_pattern_sets():
     """
     oaisets = OAISet.query.filter(OAISet.search_pattern.isnot(None))
     chain(make_oai_task_group(oais) for oais in oaisets).apply_async()
+
+
+def format_file_integrity_report(report):
+    """Format the email body for the file integrity report."""
```

Comment:
Maybe use a jinja template as for other emails? Can be done later - just create an issue for this.